packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -63,6 +63,42 @@ export interface IContainerRuntimeBaseEvents extends IEvent{
     (event: "signal", listener: (message: IInboundSignalMessage, local: boolean) => void);
 }
 
+/**
+ * Encapsulates the return codes of the aliasing API
+ */
+ export enum AliasResult {
```

Comment:
One thought since you are changing it:
enums are expensive in TS. Consider to use union type:

type AliasResult = "Success" | ...;